ext/DFTKPlotsExt.jl

```diff
@@ -108,4 +108,98 @@ function plot_dos(basis, eigenvalues; εF=nothing, unit=u"hartree",
 end
 plot_dos(scfres; kwargs...) = plot_dos(scfres.basis, scfres.eigenvalues; scfres.εF, kwargs...)
 
+function plot_ldos(basis, eigenvalues, ψ; εF=nothing, unit=u"hartree",
+                  temperature=basis.model.temperature,
+                  smearing=basis.model.smearing,
+                  εrange=default_band_εrange(eigenvalues; εF), n_points=1000, kwargs...)
+    # TODO Should also split this up into one stage doing the DOS computation
+    #      and one stage doing the DOS plotting (like now for the bands.)
+
+    n_spin = basis.model.n_spin_components
+    eshift = something(εF, 0.0)
+    εs = range(austrip.(εrange)..., length=n_points)
+
+    # Constant to convert from AU to the desired unit
+    to_unit = ustrip(auconvert(unit, 1.0))
+
+    p = Plots.plot(; kwargs...)
+    spinlabels = spin_components(basis.model)
+    colors = [:blue, :red]
+    Dεs = compute_ldos.(εs, Ref(basis), Ref(eigenvalues), Ref(ψ); smearing, temperature)
+    for σ = 1:n_spin
+        D = [Dσ[σ] for Dσ in Dεs]
+        label = n_spin > 1 ? "LDOS $(spinlabels[σ]) spin" : "LDOS"
+        Plots.plot!(p, (εs .- eshift) .* to_unit, D; label, color=colors[σ])
+    end
+    if !isnothing(εF)
+        Plots.vline!(p, [0.0], label="εF", color=:green, lw=1.5)
+    end
+
+    if isnothing(εF)
+        Plots.xlabel!(p, "eigenvalues  ($(string(unit)))")
+    else
+        Plots.xlabel!(p, "eigenvalues -ε_F  ($(string(unit)))")
+    end
+    p
 end
+plot_ldos(scfres; kwargs...) = plot_ldos(scfres.basis, scfres.eigenvalues, scfres.ψ; scfres.εF, kwargs...)
+
+function plot_pdos(p, i::Integer, l::Integer, iatom::Integer, basis, eigenvalues, ψ; εF=nothing, unit=u"hartree",
```

Comment:
To me `i` `l` and `iatom` live in the same category as `εF`. Maybe make them kwargs ? For sure in the ordering they should come ofter the basis / eigenvalues and \psi.